Lib/unittest/__init__.py

```diff
@@ -70,6 +70,37 @@ def testMultiply(self):
 # deprecated
 _TextTestResult = TextTestResult
 
+from .loader import (
+    makeSuite as _makeSuite,
```

Comment:
Why not add deprecations when they are defined?

Reply:
IIUC, it is the "shortcuts" that are deprecated, not the methods themselves. Should I deprecate the undocumented `unittest.TestLoader.makeSuite` and `unittest.TestLoader.findTestCases` as well?

Reply:
There are no `unittest.TestLoader.makeSuite` and `unittest.TestLoader.findTestCases`.

Reply:
Sorry, I meant `unittest.loader`, not `unittest.TestLoder`:

```
>>> import unittest
>>> unittest.loader.makeSuite
<function makeSuite at 0x10be0e2a0>
>>> unittest.loader.findTestCases
<function findTestCases at 0x10be0e340>
>>> 
```